.travis.yml

```diff
@@ -19,13 +21,15 @@ services:
 
 env:
   global:
-  - CORE_BRANCH=master
+  - CORE_BRANCH=stable12
   - PHP_COVERAGE=TRUE
   matrix:
   - DB=sqlite
+  - DB=pgsql
 
 branches:
   only:
+  - release
```

Comment:
Why a release branch? I usually just have one main master branch and feature branches for features and bugfixes. Releases are tagged commits on the master branch.